nyxx.interactions/lib/src/Models/SlashCommand.dart

```diff
@@ -0,0 +1,39 @@
+part of nyxx_interactions;
+
+class SlashCommand {
+
+  late final String name;
+  late final String description;
+  late final Cacheable<Snowflake, Guild>? guild;
+  late final List<CommandArg> args;
+  late final bool isGlobal;
+  late bool isRegistered = false;
+
+  late final Nyxx _client;
+
+  SlashCommand._new(this._client, this.name, this.description, this.args, {this.guild}) {
+    this.isGlobal = guild == null;
+  }
+
+  Future<SlashCommand> _register() async {
+    final options = List<Map<String, dynamic>>.generate(
+        this.args.length, (i) => this.args[i]._build());
```

Comment:
map on existing collection instead of generate

Reply:
Done!